src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -124,41 +124,39 @@ public async Task StopAsync(CancellationToken cancellationToken)
             {
                 try
                 {
-                    if (!_started)
+                    if (_started)

```

Comment:
How did it work before? Did we never stop the started listener?

Reply:
There is a flag in check in [FunctionListener](https://github.com/Azure/azure-webjobs-sdk/blob/cc7522d2ea1d1c2ac667ab7490fa37460e4af53a/src/Microsoft.Azure.WebJobs.Host/Listeners/FunctionListener.cs#L139) that wraps this listener so from WebJobs/Functions side it will not call this twice (and encounter the exception).  Now we are calling it from Cancel so its important to not throw an exception here.  Its also consistent with other extensions.